trainingMaquetado/maquetado.css

```diff
@@ -6,6 +6,30 @@
   margin-bottom: 20px;
 }
 
+.subscribe {
+  background-color: Transparent;
+  border-color: white;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 0.5px;
+  color: white;
+  font-size: 16px;
+  font-weight: bold;
+  height: 40px;
+  margin-bottom: 20px;
+  width: 120px;  
```

Comment:
don't set a fixed width, let it adapt to the content and use a padding to separate the element from its borders.